packages/transition-backend/src/services/transitRouting/TrRoutingBatch.ts

```diff
@@ -324,22 +299,21 @@ class TrRoutingBatch {
         pathCollection?: PathCollection;
     }> => {
         const resultHandler = createRoutingFileResultProcessor(
-            this.options.absoluteBaseDirectory,
-            this.demandParameters,
-            this.batchRoutingQueryAttributes,
-            this.options.inputFileName
+            this.job.getJobFileDirectory(),
+            this.job.attributes.data.parameters.demandAttributes.configuration,
+            this.job.attributes.data.parameters.transitRoutingAttributes,
+            this.job.getInputFileName()
```

Comment:
Should you pass the job to the fileResultProcessor? Instead of passing all the individual parameters, also see with the comment to use `getFilePath` from the job, it may be useful to do so in that class too.